### PR TITLE
3284 disk image block size

### PR DESCRIPTION
**Pull request by dgrove727**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/casemodule/AddImageTask.java [hunk 1]
```diff
@@ -126,9 +129,6 @@ public void run() {
             Thread progressUpdateThread = new Thread(new ProgressUpdater(progressMonitor, tskAddImageProcess));
             progressUpdateThread.start();
             runAddImageProcess(errorMessages);
-            if (null != progressUpdateThread) {
-                progressUpdateThread.interrupt();
-            }
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/casemodule/AddImageTask.java]: Why was this removed? It seems unrelated to sector size. Was it broken?

Comment (dgrove727) [reply Core/src/org/sleuthkit/autopsy/casemodule/AddImageTask.java]: The reason I took this out was because, as the IDE indicates, 'progressUpdateThread' is never going to be null at this point. But in re-examining this again, I realize now that the thread interrupt should still be there, because the condition check is for *not* null. So I'll get a fix for this.
